addons/core/addon/components/session-status/index.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@glimmer/component';
+
+const STATUS_MAPPING = {
+  //map session status to hds text color and icon style
+  //more info - https://design-system-components-hashicorp.vercel.app/components/badge
+  active: ['success', 'check'],
+  pending: ['neutral', 'delay'],
+  canceling: ['critical', 'x'],
+  terminated: ['critical', 'x'],
+};
+
+export default class SessionStatus extends Component {
+  get statusColor() {
+    return STATUS_MAPPING[this.args.status][0];
+  }
+
+  get statusIcon() {
+    return STATUS_MAPPING[this.args.status][1];
+  }
+}
```

Comment:
Non-blocking question
Should we set some kind of default here? Just thinking from the aspect of handling unknown credential types, we could try for the same here. Since we rely on the status from the BE, technically they could add a new session status that we have not accounted for. However, this may not be an issue since we will release everything in parody now. I included a status of `stale` and it shows no badge as an example.
<img width="1088" alt="Screen Shot 2022-10-26 at 9 25 51 AM" src="https://user-images.githubusercontent.com/29386339/198054108-ed9d54d1-1fa8-4d54-a81f-e5aa84ef46ac.png">

Reply:
Ah, Good point. I'm unsure what sort of `default` status we should set. There is scope for adding more types with credentials, but I think there may not be new types for `status.` Either way, I'll raise this question in the channel

Reply:
We should show status strings even when we don't have styles for them, and default them to a neutral badge with no icon.